src/spikeinterface/widgets/unit_summary.py

```diff
@@ -107,15 +107,18 @@ def plot_matplotlib(self, data_plot, **backend_kwargs):
         # and use custum grid spec
         fig = self.figure
         nrows = 2
-        ncols = 3
+        ncols = 2
```

Comment:
It might be worth expanding the docs just to briefly introduce what things are plot and under what conditions. For example are some plots conditional on the steps run while others will always be plot? I got a little confused here because `unit_locations` is under a conditional but it's inclusion doesn't add to the number `ncols`. Maybe the initial `ncols=2` is reserved for `unit_locations`? but then I don't think `unit_locations` need to be under the conditional if it will always be plot. I tried skipping unit locations and there was some unused space in the plot so maybe this is the case 🤔

Reply:
(I just read the diff more carefully and see this is not actually related to your changes in this PR, sorry! but maybe you know the answer

Reply:
Hello - just noticed that at the moment the `export_report` function computes `unit_locations` if they're not already computed (line 90 in `report.py`). So the first plot also always appears.

So either:
- remove the conditional on the unit locations plot
- remove the automatic computation

Opinions @samuelgarcia @zm711 ?
I'd vote to remove the automatic computation, since I don't think we should be computing anything in a plotting function.

Reply:
Sounds ok to me, Zach! I've updated to remove the conditional on the unit locations plot.